testsuites/store/src/main/resources/testcases/rdf-mt-inferencing/type/test003-in.nt

```diff
@@ -5,4 +5,6 @@
 # Relevant inferencing rules: rdfs10
 # Author: Arjohn Kampman (arjohn.kampman@aidministrator.nl)
 <http://example.org/prop> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
-_:prop <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
+
+# this entails that _:prop must be allowed as a predicate, but that is not legal in RDF
+#_:prop <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#ContainerMembershipProperty>.
```

Comment:
True, but that doesn't mean that _this_  isn't allowed. The inferred triples in this particular test are valid, even if you can never actually use `_:prop` as a predicate.